diff -durN inetlib-1.1.1.orig/Makefile.am inetlib-1.1.1/Makefile.am
--- inetlib-1.1.1.orig/Makefile.am	2005-06-06 23:00:24.000000000 +0200
+++ inetlib-1.1.1/Makefile.am	2006-05-18 15:03:23.000000000 +0200
@@ -280,7 +280,7 @@
 	find $(dst) -name '*.class' -exec rm -f '{}' ';'
 
 install-data-local:
-	$(mkinstalldirs) $(prefix)/share/java
-	$(INSTALL_DATA) $(inetlib_jar) $(prefix)/share/java
+	$(mkinstalldirs) $(DESTDIR)$(prefix)/share/java
+	$(INSTALL_DATA) $(inetlib_jar) $(DESTDIR)$(prefix)/share/java
 
 # End of Makefile.am
